Documentation/spec/metadata_format_spec.md

```diff
@@ -238,7 +237,21 @@ You **SHALL NOT** define new *item* in markdown.
 
 ### 3.2 Reference Items in Markdown
 
-We introduce a new markdown syntax to represent *item* reference:
+To cross reference an *item*, you can use URL with `xref` scheme. For example, to cross reference `System.String`: 
```

Comment:
According to <https://en.wikipedia.org/wiki/Uniform_Resource_Locator>, URL is more accurate here?